src/main/java/ru/javawebinar/topjava/web/MealServlet.java

```diff
@@ -0,0 +1,84 @@
+package ru.javawebinar.topjava.web;
+
+import ru.javawebinar.topjava.dao.MealDaoImplMem;
+import ru.javawebinar.topjava.model.Meal;
+import ru.javawebinar.topjava.model.MealWithExceed;
+import ru.javawebinar.topjava.service.MealService;
+import ru.javawebinar.topjava.service.MealServiceImplMem;
+import ru.javawebinar.topjava.util.MealsUtil;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import org.slf4j.Logger;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class MealServlet extends HttpServlet {
+    private static final Logger LOG = getLogger(MealServlet.class);
+    
+    
+    private MealService mealService;
+
+    @Override
+    public void init() throws ServletException {
+        super.init();
+        mealService = new MealServiceImplMem();
+    }
+
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        request.setCharacterEncoding("UTF-8");
+
+        String sId = request.getParameter("id");
+
+        Meal meal = new Meal(sId.isEmpty() ? null : Integer.valueOf(sId),
+                LocalDateTime.parse(request.getParameter("dateTime")),
+                request.getParameter("description"),
+                Integer.valueOf(request.getParameter("calories")));
+        mealService.saveMeal(meal);
+        LOG.debug("meal" + meal + " saved");
```

Comment:
In Log your meal string will looks like: mealMeal@5464356363 saved.
1) You should make offset from "meal"
2) meal object should be invoke toString() method